content/en/real_user_monitoring/mobile_and_tv_monitoring/troubleshooting/ios.md

```diff
@@ -84,6 +84,10 @@ private class YourCustomDelegateURLSessionDelegate: NSObject, URLSessionTaskDele
 * implement `URLSessionDataDelegate` and forward:
   * [`urlSession(_:dataTask:didReceive:)`][6]
 
+## "Deobfuscation failed" warning
+
+A warning appears when symbolication fails for a stack trace. If the stack trace is not symbolicated to begin with, you can ignore this warning. Otherwise, use the [RUM Debug Symbols page][7] to view all your uploaded dSYMs. See [Investigate Obfuscated Stack Traces with RUM Debug Symbols][8].
```

Comment:
Just confirming that we are intentionally using `symbolication` instead of `deobfuscation` here, feel free to disregard the comment if we are!

```suggestion
A warning appears when deobfuscation fails for a stack trace. If the stack trace is not deobfuscation to begin with, you can ignore this warning. Otherwise, use the [RUM Debug Symbols page][7] to view all your uploaded dSYMs. See [Investigate Obfuscated Stack Traces with RUM Debug Symbols][8].
```